discussions/api/v1/views.py

```diff
@@ -15,3 +21,16 @@ class UserDetailView(generics.RetrieveAPIView):
     # shouldn't be necessary in theory but DRF is returning Not Found otherwise
     def get_object(self):
         return User.objects.get(external_id=self.kwargs['external_id'])
+
+
+class SubscriptionsDetailView(generics.RetrieveAPIView):
```

Comment:
so actually this interface isn't part of the cs_comments_service API - see https://github.com/edx/cs_comments_service/blob/master/api/notifications_and_subscriptions.rb#L21-L27

it's only POST or DELETE and the user_id from the URL becomes the subscriber_id.  (i know, weird)